packages/react-core/rollup.config.mjs

```diff
@@ -15,7 +15,7 @@ const config = defineConfig([
   // CJS config
   {
     input,
-    output: { dir: 'dist', format: 'cjs' },
+    output: { dir: 'dist', esModule: true, format: 'cjs' },
```

Comment:
This change seems tangential, is it intended?

Reply:
It's intentional, there's more coming